src/test/java/calculator/FormulaTest.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class FormulaTest {
+
+    @Test
+    void 입력값_NULL_예외() {
+        assertThatThrownBy(() -> {
+            Formula formula = new Formula(null);
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
```suggestion
    void 입력값_NULL_예외() {
        assertThatThrownBy(() -> new Formula(null))
                .isInstanceOf(IllegalArgumentException.class);
    }
```

formula 변수는 사용하지 않으니 좀 더 간단하게 작성할 수 있을것 같아요 :)